projects/packages/my-jetpack/_inc/components/welcome-banner/index.jsx

```diff
@@ -0,0 +1,78 @@
+import { Container, Col, Button, Text } from '@automattic/jetpack-components';
+import { __ } from '@wordpress/i18n';
+import { close } from '@wordpress/icons';
+import { useEffect, useCallback, useState } from 'react';
+import useAnalytics from '../../hooks/use-analytics';
+import useWelcomeBanner from '../../hooks/use-welcome-banner';
+import { CardWrapper } from '../card';
+import styles from './style.module.scss';
+
+/**
+ * Component that renders the Welcome banner on My Jetpack.
+ *
+ * @returns {object} The WelcomeBanner component.
+ */
+const WelcomeBanner = () => {
+	const { recordEvent } = useAnalytics();
+	const { hasBeenDismissed, dismissWelcomeBanner } = useWelcomeBanner();
+	const [ bannerVisible, setBannerVisible ] = useState( ! hasBeenDismissed );
+
+	useEffect( () => {
+		if ( bannerVisible ) {
+			recordEvent( 'jetpack_myjetpack_welcome_banner_view' );
```

Comment:
This seems to be triggered a lot of times when the page is initially rendered; maybe we should add some dependencies to avoid triggering it so much?


https://github.com/Automattic/jetpack/assets/426388/9edb7b60-cbd4-4394-b7df-b3ce6c66ed53